src/ingest_llm_as/api/repository.py

```
@@ -71,6 +81,45 @@ async def ingest_python_repository(
+                "include_patterns": request.include_patterns,
+                "exclude_patterns": request.exclude_patterns[:METADATA_LIST_TRUNCATION_LIMIT],  # First 5 for brevity
+            },
+            captured_at=datetime.utcnow(),
+            processed=False,
+        )
+        db.add(raw_record)
+        db.commit()
+        
+        logger.info(f"Raw repository ingestion persisted: {ingestion_id} ({request.source_path})")
+        
+    except IntegrityError as e:
```

Ruff (0.14.11)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)